### ISSUE TITLE
person_test.UT_IsSame.test_05_different_hub FAILS

### ISSUE THREAD

**Issue by djfkahn** [djfkahn/MemberHubDirectoryTools#99]
```
WHERE:
	 test_05_different_hub (person_test.UT_IsSame)
WHAT:
 Traceback (most recent call last):
  File "/Users/djfkahn/Desktop/DEV/MemberHubDirectoryTools/TEST/person_test.py", line 83, in test_05_different_hub
    self.assertFalse(person_A.IsSame(person_B))
AssertionError: True is not false
```

**Comment by djfkahn** [comment]
This is related to issue #88. Need to mark this as an expected Failure.

**Comment by djfkahn** [comment]
addressed in 50a7d96
